keyboards/westfoxtrot/aanzee/aanzee.c

```diff
@@ -0,0 +1,52 @@
+/* Copyright 2018 westfoxtrot
+
+*
+
+* This program is free software: you can redistribute it and/or modify
+
+* it under the terms of the GNU General Public License as published by
+
+* the Free Software Foundation, either version 2 of the License, or
+
+* (at your option) any later version.
+
+*
+
+* This program is distributed in the hope that it will be useful,
+
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+
+* GNU General Public License for more details.
+
+*
+
+* You should have received a copy of the GNU General Public License
+
+* along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+*/
+
+#include "aanzee.h"
+
+
+void led_set_kb(uint8_t usb_led) {
+
+if (usb_led & (1<<USB_LED_CAPS_LOCK)) {
+
+// Turn capslock on
+
+PORTB &= ~(1<<2);
```

Comment:
If possible, we should start using the GPIO commands here, since these are hardware agnostic:

So I believe that this would be: 
```suggestion
  writePinLow(B2);
```